src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -303,11 +303,12 @@ export default class AircraftController {
 
     /**
      * @for AircraftController
-     * @method aircraft_visible
+     * @method isAircraftVisible
      * @param aircraft {AircraftModel}
      * @param factor {number}
+     * @returns {boolean}
      */
-    aircraft_visible(aircraft, factor = 1) {
+    isAircraftVisible(aircraft, factor = 1) {
         return vlen(aircraft.relativePosition) < AirportController.airport_get().ctr_radius * factor;
```

Comment:
Like the new method name 😄 

Since we're here, let's change this to:
```js
const visibleDistance = AirportController.airport_get().ctr_radius * factor;

           // km              // km
return aircraft.distance < visibleDistance;
```
Also note that there are some comments and an enum `DEFAULT_CTR_RADIUS_NM` which are incorrectly labeled, as the `#distance` property is actually in kilometers. And we can save here (slightly) by using `#distance` over `vlen(aircraft.relativePosition)` since it is calculated every frame already.

Reply:
done